### ISSUE TITLE
Add npmjs package for azure-maps-html-marker-layer

### ISSUE THREAD

**Issue by mdelrossi1** [Azure-Samples/azure-maps-html-marker-layer#2]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: 

- [ ] bug report -> please search issues before submitting
- [x] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)

### Minimal steps to reproduce
> Attempt to use this in an npm application

### Expected/desired behavior
> I would like to cleanly use this along with the `azure-maps-control` npm package in a react application.

### Versions
> "0.0.1"

### Mention any other details that might be useful

> I'm not sure if this is the best place for this, but it would be awesome if we could use npm to install this as a package.  Adding to our `dist` folder is a little brittle for us, might cause confusion for future developers, and does not effectively handle versioning for a feature like this.  Currently, we have to lift and shift this code to our own repository, which isn't ideal either.  Having an installable package in npmjs would clarify this dependency and make future maintenance easier.

**Comment by rbnrs** [comment]
Yes, it would be nice if you can add this to a npm registry so it can be install with the map control package

**Comment by mitikov** [comment]
@rbrundritt Could we please get this one rolling?
Thanks.

**Comment by mitikov** [comment]
Hey @mdelrossi1, @rbnrs folks, 

I'm wondering how are things for you with this PBI, a need to have NPM package for `azure-maps-html-marker-layer`.

I'm struggling with bringing both `azure-maps-control`, `storybook` and HtmlMarker layer; was wondering if any of you ever made it thru?

Thanks,
Nik

**Comment by rbrundritt** [comment]
I'm no longer on the Azure Maps team and there is an internal process for rolling out packages like this at Microsoft where this should be done by the Azure Maps dev team. This was handed off to that team and was planned a while back, but not sure where they are with this now. I'll ping the team internally to see if they have any updates on this.
